book/_toc.yml

```diff
@@ -14,3 +14,4 @@ parts:
   - file: notebooks
   - file: markdown-notebooks
   - file: tut01_firstfigure
+  - file: tut02_spec_pd_gpd
```

Comment:
Maybe we should use "spe" instead of "spec" as the later one seems to refere to scientific Python ecosystem **coordination** (please see https://scientific-python.org/specs/ [, and Dongdong's very detailed review on my draft for the pygmt talk])?